app/controllers/stages_controller.rb

```diff
@@ -1,14 +1,14 @@
 class StagesController < ApplicationController
   def create
     @stage = Stage.new(stages_params)
-    
+
     if @stage.save
-      redirect_to(:controller => 'guides', 
-        :action => 'edit', 
+      redirect_to(:controller => 'guides',
+        :action => 'edit',
         :id => @stage.guide.id)
     else
-      redirect_to(:controller => 'guides', 
-        :action => 'edit', 
+      redirect_to(:controller => 'guides',
+        :action => 'edit',
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.